javascript/javascript1/week1/index.js

```diff
@@ -12,7 +13,7 @@ dogYearFuture = 2027;
 dogYear = dogYearFuture - dogYearOfBirth;
 let shouldShowResultInDogYears = dogYear;
 
-if (shouldShowResultInDogYears!==true) {
+if (shouldShowResultInDogYears !== true) {
   console.log(
     `Your dog will be ${dogYear * 7} dog years old in ${dogYearFuture}`
   );
```

Comment:
I can't see you define the variables like yearFuture, age, dogYear. And use const for variables that don't change eg const yearOfBirth = 1987;

Reply:
I think i forgot to assign them:
const dogYearOfBirth = 2017;
const dogYearFuture = 2027;
let dogYear = dogYearFuture - dogYearOfBirth;